manifest.json

```diff
@@ -2,9 +2,9 @@
   "id": "nldates-obsidian",
   "name": "Natural Language Dates",
   "description": "Create date-links based on natural language",
-  "version": "0.5.2",
+  "version": "0.6.0",
   "author": "Argentina Ortega Sainz",
-  "authorUrl": "",
+  "authorUrl": "https://argentinaos.com/",

```

Comment:
Let me know if you're okay with linking to your website here. Just saw it was missing 😄